java/test/jmri/jmrix/internal/configurexml/load/SampleSensorsAndTurnouts.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="/xml/XSLT/panelfile-2-9-6.xsl" type="text/xsl"?>
+<layout-config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/layout-2-9-6.xsd">
+  <jmriversion>
+    <major>4</major>
+    <minor>16</minor>
+    <test>0</test>
+    <modifier>ish</modifier>
+  </jmriversion>
+  <sensors class="jmri.jmrix.internal.configurexml.InternalSensorManagerXml">
+    <defaultInitialState>unknown</defaultInitialState>
+    <sensor inverted="false">
+      <systemName>ISCLOCKRUNNING</systemName>
+      <userName>Clock</userName>
+    </sensor>
+    <sensor inverted="false">
+      <systemName>ISBONDALE JCTN GOODS</systemName>
+    </sensor>
+    <sensor inverted="false">
+      <systemName>ITCASETEST</systemName>
```

Comment:
Does this Sensor deliberately have an incorrect type letter (_T_ instead of _S_)?

Reply:
This incorrect type letter is the *only* test failure in rhwood/JMRI#20

Reply:
No, that's a typo.  Will fix. Thanks for catching.